lib/id3v2/ID3v24TagMapper.ts

```diff
@@ -190,7 +190,7 @@ export class ID3v24TagMapper extends CaseInsensitiveTagMap {
           case 'AverageLevel':
           case 'PeakValue':
             tag.id += ':' + tag.value.owner_identifier;
-            tag.value = tag.value.data.length === 4 ? tag.value.data.readUInt32LE(0) : null;
+            tag.value = tag.value.data.length === 4 ? new DataView(tag.value.data.buffer).getUint32(0) : null;
```

Comment:
The original read was little endian so you need to pass `true` to `getUint32`.

```suggestion
            tag.value = tag.value.data.length === 4 ? new DataView(tag.value.data.buffer).getUint32(0, true) : null;
```